client/src/main/scala/com/dwolla/cloudflare/domain/model/ratelimits/RateLimitAction.scala

```diff
@@ -1,10 +1,21 @@
 package com.dwolla.cloudflare.domain.model.ratelimits
 
-case class RateLimitAction (
-  mode: String,
-  timeout: Int,
-  response: Option[RateLimitActionResponse] = None
-)
+sealed trait RateLimitAction {
+  val mode: String
+  val response: Option[RateLimitActionResponse]
+}
+
+case class TimeoutRateLimitAction(mode: String, timeout: Int, response: Option[RateLimitActionResponse] = None) extends RateLimitAction
```

Comment:
Thinking about the documented cases in Cloudflare… is it worth splitting this into two cases in the ADT, specifically for "ban" and "simulate", instead of letting `mode` be any string?